docs/TASKS.md

```diff
@@ -176,8 +177,15 @@ Snap is designed in a way where custom schedulers can easily be dropped in. If a
       "max-failures": 10,
    ```
   
-    
-    
+##### Streaming Schedule
+```yaml
+   ---
+  version: 1
+  schedule:
+    type: "streaming"
+```
+For more details on streaming, visit [STREAMING.md](STREAMING.md)
```

Comment:
Could you add a sentence that there is no interval and count - those params will be skipped if they are provided